### PR TITLE
Search builder by tags on aws and gce

### PR DESCRIPTION
**Pull request by aleksbykov**
Static ip addresses are removed from code.
Now all builders will be looked by tag nodetype:builder
on aws and gce

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/utils/common.py [hunk 1]
```diff
@@ -1148,38 +1148,53 @@ def filter_gce_instances_by_type(instances):
...
+        })
+
+    return builders
+
+
+def get_gce_builders():
+    builders = []
+    ssh_key_path = os.path.join(SSH_KEY_DIR, SSH_KEY_GCE_DEFAULT)
+
+    gce_builders = list_instances_gce(tags_dict={"nodetype": "builder"})
```
Comment (bentsi) [inline sdcm/utils/common.py]: why `nodetype` is different from AWS `NodeType`?

File: sdcm/utils/common.py [hunk 2]
```diff
@@ -1148,38 +1148,53 @@ def filter_gce_instances_by_type(instances):
...
+
+    for gce_builder in gce_builders:
+        builders.append({
+            "public_ip": gce_builder.public_ips[0],
+            "name": gce_builder.name,
+            "user": "scylla-test",
```
Comment (bentsi) [inline sdcm/utils/common.py]: builder user is `jenkins`.
 `scylla-test` is used by SCT

Comment (aleksbykov) [reply sdcm/utils/common.py]: i will recheck, but i can't get access to gce instances with jenkins user and gce key

Comment (aleksbykov) [reply sdcm/utils/common.py]: @bentsi  can't get access with jenkins ans scylla-test key
```
abykov@ububuntu:~/Projects/scylla-cluster-tests$ ssh -i ~/.ssh/scylla-test jenkins@35.229.76.4
jenkins@35.229.76.4: Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
abykov@ububuntu:~/Projects/scylla-cluster-tests$ ssh -i ~/.ssh/scylla-qa-ec2 ### jenkins@35.229.76.4
jenkins@35.229.76.4: Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
abykov@ububuntu:~/Projects/scylla-cluster-tests$ ssh -i ~/.ssh/scylla-test jenkins@35.229.76.4
jenkins@35.229.76.4: Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
abykov@ububuntu:~/Projects/scylla-cluster-tests$ ssh -i ~/.ssh/scylla-test scylla-test@35.229.76.4
Last login: Fri Nov  8 22:11:13 2019 from 50.233.106.94
[scylla-test@public-jenkins-builder4-qavpc ~]$ exit
logout
Connection to 35.229.76.4 closed.
```

Comment (fruch) [reply sdcm/utils/common.py]: use scylla-test user

Comment (fruch) [reply sdcm/utils/common.py]: i.e. the jenkins for some reason on gce has it's own key

Comment (bentsi) [reply sdcm/utils/common.py]: For some reason, it works only for builder 1..., let's keep it like you implemented.

File: sdcm/utils/common.py (hunk 1 above)
Comment (bentsi) [reply sdcm/utils/common.py]: I changed the tags on the Gce builders. Just change in the code to `NodeType`

Comment (aleksbykov) [reply sdcm/utils/common.py]: resolved

**Comment by bentsi** [comment]
please resolve conflicts

**Comment by aleksbykov** [comment]
@bentsi  resolved

File: sdcm/utils/common.py (hunk 1 above)
Comment (bentsi) [reply sdcm/utils/common.py]: I don't see it is resolved...
image

**Comment by bentsi** [comment]
Please don't click on "Resolve" button since I need to verify it is resolved and how

File: sdcm/utils/common.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/utils/common.py]: Fixed

**Comment by aleksbykov** [comment]
@bentsi resolved

**Pull request merged**